staging/src/kubevirt.io/client-go/kubecli/replicaset.go

```diff
@@ -156,15 +121,7 @@ func (v *rc) PatchStatus(name string, pt types.PatchType, data []byte) (result *
 }
 
 func (v *rc) UpdateStatus(vmi *v1.VirtualMachineInstanceReplicaSet) (result *v1.VirtualMachineInstanceReplicaSet, err error) {
-	result = &v1.VirtualMachineInstanceReplicaSet{}
```

Comment:
Missed `PatchStatus` above?

Reply:
There isn't a `PatchStatus` method in the replicaset interface :)

Reply:
I think you want to use `subresources` argument of the `Patch`.

Reply:
@fossedihelm

Reply:
This is a good idea! Thank you!